### PR TITLE
Fixing windows start jupyter instructions

### PR DESCRIPTION
**Pull request by patrick-nicholson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ below.
 
 ```bash
 # switches to your home directory
-cd
+cd %userprofile%
 # activates the conda environment you installed
-datafest\Scripts\activate.bat
+datafest\Scripts\activate
```
Comment (parente) [inline README.md:56]: On my windows 7 box, there was no "activate" only "activate.bat". Yours didn't have the .bat?

Comment (patrick-nicholson) [reply README.md:56]: I actually have both, my prompt autocompleted to `activate` so I didn't see `activate.bat`.

Comment (parente) [reply README.md:56]: Hmm. Weird.

Comment (parente) [reply README.md:56]: I installed again and see both now. Let's go with yours.

**Pull request merged**